packages/wonder-blocks-modal/components/modal-close-button.test.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import React from "react";
+import {shallow} from "enzyme";
+
+import ModalCloseButton from "./modal-close-button.js";
+
+describe("ModalCloseButton", () => {
+    test("Clicking the button triggers `onClick`", () => {
```

Comment:
Probably not necessary b/c it's placeholder component.

Reply:
As discussed in another comment, we're planning to keep it! 👍🏻